ee/clickhouse/views/groups.py

```diff
@@ -18,11 +18,48 @@ class Meta:
         fields = ["group_type", "group_type_index"]
 
 
-class ClickhouseGroupsView(StructuredViewSetMixin, ListModelMixin, viewsets.GenericViewSet):
+class ClickhouseGroupsTypesView(StructuredViewSetMixin, ListModelMixin, viewsets.GenericViewSet):
     serializer_class = GroupTypeSerializer
     queryset = GroupTypeMapping.objects.all()
     pagination_class = None
 
+
+class ClickhouseGroupsView(StructuredViewSetMixin, ListModelMixin, viewsets.GenericViewSet):
+    serializer_class = ClickhouseGroupSerializer
+    queryset = None
+    pagination_class = None
+
+    def list(self, request, *args, **kwargs):
+        limit = 100
+        offset = int(request.GET.get("offset", 0))
+
+        query_result = sync_execute(
+            """
+                SELECT group_type_index, group_key, created_at, group_properties FROM groups
+                INNER JOIN (
+                    SELECT
+                        group_key,
+                        max(created_at) as created_at
+                    FROM groups
+                    WHERE team_id = %(team_id)s AND group_type_index = %(group_type_index)s
+                    GROUP BY group_key
+                    LIMIT 100 OFFSET %(offset)s
+                ) latest_groups
```

Comment:
The query approach here is wrong. As I mentioned in the latest pairing session, the approach is `group by` and 
 `argMax` instead.